lib/service/storage/elasticsearch.ts

```diff
@@ -1124,11 +1259,19 @@ class ElasticSearch extends Service {
    * @returns {Promise.<{ successes: [_id, _source, _status], errors: [ document, status, reason ] }>}
    */
   async updateByQuery(
-    index,
-    collection,
-    query,
-    changes,
-    { refresh, size = 1000, userId = null } = {},
+    index: string,
+    collection: string,
+    query: JSONObject,
+    changes: JSONObject,
+    {
+      refresh,
+      size = 1000,
+      userId = null,
+    }: {
+      refresh?: boolean | "wait_for";
+      size?: number;
+      userId?: string;
+    } = {},
```

Comment:
```suggestion
    {
      refresh,
      size = 1000,
      userId = null,
    }: Pick<KRequestParams, "refresh" | "userId"> & {
      size?: number;
    } = {},
```

With new `KRequestParams` type we can compose type like it, to DRY